components/Alert/Alert.stories.jsx

```diff
@@ -1,8 +1,8 @@
-import { Alert } from '.'
+import { Header } from '.'
```

Comment:
Should this file be changed?

Reply:
Second this. Should the file be called Header instead of Alert?